nautobot/utilities/tests/test_forms.py

```diff
@@ -429,3 +431,52 @@ def test_invalid_input(self):
         for test in tests:
             with self.assertRaises(forms.ValidationError):
                 self.field.clean(test)
+
+
+class AddressFieldMixinTest(TestCase):
+    """Test cases for the AddressFieldMixin."""
+
+    def setUp(self):
+        """Setting up shared variables for the AddressFieldMixin."""
+        self.dummy_address = "10.0.0.1/24"
+        self.initial = {"address": self.dummy_address}
+
+    def test_address_initial(self):
+        """Ensure initial kwargs for address is passed in."""
+        with mock.patch("nautobot.utilities.forms.forms.forms.ModelForm.__init__") as mock_init:
```

Comment:
I had to look this path up myself to be sure it was correct. :-) Maybe add a comment along the lines of:

`# Mock the django.forms.ModelForm.__init__ method as imported into nautobot.utilities.forms.forms`

Reply:
Sure.